packages/dcos-history/extra/history/server.py

```diff
@@ -90,4 +92,10 @@ def start():
 
     state_buffer = BufferCollection(os.environ['HISTORY_BUFFER_DIR'])
     BufferUpdater(state_buffer, headers_cb).run()
-    app.run(host='0.0.0.0', port=int(os.environ.get('PORT', '15055')))
+
+
+def start():
```

Comment:
The "needs to be in the right path" thing is relatively easily fixable by just changing the gunicorn exec to not be an absolute path. then a `pip install -e` into a virtualenv makes it easy to edit / work interactively.